apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v1/metadata.json

```diff
@@ -130,16 +130,6 @@
         "type" : "string",
         "documentation" : "The offset used for result pagination"
       } ]
-    }, {
-      "type" : "StructureDefinition",
-      "profile" : {
-        "reference" : "http://hl7.org/fhir/Profile/StructureDefinition"
-      },
-      "interaction" : [ {
-        "code" : "read"
-      }, {
-        "code" : "search-type"
-      } ]
```

Comment:
I believe this was removed in HAPI 5.1.0 with this entry in the release notes (https://hapifhir.io/hapi-fhir/docs/introduction/changelog_2020.html):

A very old feature that is not believed to be used anywhere has been removed: The ServerProfileProvider is a special resource provider that was automatically registered to HAPI FHIR REST servers, and served up StructureDefinitions that were registered to the FhirContext. Registering custom StructureDefinitions against the FhirContext for exposure through the REST API (as what was then the /Profile endpoint) was planned to be a common feature during the DSTU1 lifecycle but did not turn out to be a useful approach. This feature was mostly forgotten about until the logic for selecting resource provider handler methods was revamped and the old mechanism suddenly became the default resource provider for StructureDefinition resources in the JPA server. We don't expect any negative impact by this change, please post in our mailing list if you disagree.

On my local server without these HAPI upgrades I see this when querying the StructureDefinition:

(master) [ bfd-server ] % curl --silent --insecure --cert $BFD_PATH/apps/bfd-server/dev/ssl-stores/client-unsecured.pem 'https://localhost:9953/v2/fhir/StructureDefinition'
{"resourceType":"Bundle","id":"98f1e17f-8d1f-42aa-aa7c-8e5c2172ad1a","meta":{"lastUpdated":"2022-04-04T20:43:30.265+00:00"},"type":"searchset","total":1,"link":[{"relation":"self","url":"https://localhost:9953/v2/fhir/StructureDefinition"}],"entry":[{"fullUrl":"https://localhost:9953/v2/fhir/StructureDefinition/operationdefinition","resource":{"resourceType":"StructureDefinition","id":"operationdefinition"}}]}%                                                                     (master) [ bfd-server ] % 

which looks similar to what was reported in the defect:

https://github.com/hapifhir/hapi-fhir/issues/1855

Reply:
fyi @karlmdavis